packages/sync-actions/package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "fast-equals": "1.6.0",
-    "jsondiffpatch": "0.2.4",
+    "jsondiffpatch": "0.3.11",
```

Comment:
oooh... i don't trust this lib enough to allow a minor upgrade anymore tbh..

Reply:
Looking at the repo, there is not a changelog either. :/

Reply:
Hmm, fair point. Only thing I can find is that there are no breaking changes in this update. `0.3.x` only dropped support for old node versions. 
More reading here: https://github.com/benjamine/jsondiffpatch/issues/213

`0.2.4` dates back to September 2016... I'd personally prefer to go through with the update or find a new, better maintained, library to use instead.

Reply:
> I'd personally prefer to go through with the update or find a new, better maintained, library to use instead

👍 
if there is time to evaluate other options, we definitely should do that.

Reply:
Sounds good. How about we go through with this one and create a new issue for replacing the library? Then make sure to get it into the planning moving forward.